app/app.js

```diff
@@ -318,7 +321,10 @@ class MaterialColors {
     let menu = Menu.buildFromTemplate(
         formats.map(format => ({
           label: `Copy ${format}`,
-          click: () => electron.clipboard.writeText(format)
+          click: () => {
+              electron.clipboard.writeText(format);
```

Comment:
for future ref, mind using 2 spaces for indent, except 4 spaces for continuations of previous lines? thx!